### ISSUE TITLE
YSF prevents sampctl from reading logprintf lines from stdout

### ISSUE THREAD

**Issue by kristoisberg** [Southclaws/sampctl#223]
We had a discussion about this with @Southclaws and @IllidanS4 some time ago. [The way YSF hooks logprintf](https://github.com/IllidanS4/YSF/blob/master/src/Hooks.cpp#L311) somehow prevents sampctl from reading `logprintf` lines from `stdout`.

**Comment by Southclaws** [comment]
Yes I forgot about this, I need to make it read stderr as well as stdout.

However, this is an issue with YSF not being backwards compatible - it should not change the output stream of `logprintf`, it should write to stdout.

**Comment by IllidanS4** [comment]
I agree `logprintf` should behave like normal with YSF, I have no issue with it being changed back (feel free to make a pull request). You should read stderr though as well, since for example C++ exceptions are reported there.

**Comment by Southclaws** [comment]
Does this happen on linux or windows?

I'm using [pty](https://github.com/kr/pty) for the terminal output on linux and according to the source, it's capturing stderr and stdout.

On Windows, stderr is also being captured: https://github.com/Southclaws/sampctl/blob/master/runtime/run_windows.go#L11
